DB update caution: Do not force it (#1087)
ezsystems/developer-documentation: docs/updating/4_update_database.md

    Always back up your data before running any database update scripts.

    After updating the database, clear the cache.
    

>     Do not use `--force` on update queries. If there is any problem applying the updates, it is best if the query fails immediately, so you can fix the underlying problem before trying to execute the update again. If you leave this for later, you risk ending up with an incompatible database, and larger problems further down the road.

Reviewer: ```suggestion
    Do not use `--force` argument for `mysql` / `psql` when performing update queries. If there is any problem applying the updates, it is best if the query fails immediately, so you can fix the underlying problem before trying to execute the update again. If you leave this for later, you risk ending up with an incompatible database that might not surface before further down the road.
```

Author: Ok, good, but then I'll tune the last sentence a bit further.

>     Do not use `--force` argument for `mysql` / `psql` commands when performing update queries. If there is any problem applying the updates, it is best if the query fails immediately, so you can fix the underlying problem before trying to execute the update again. If you leave this for later you risk ending up with an incompatible database, though the problems might not surface until later.

Reviewer 2: ```suggestion
    Do not use `--force` argument for `mysql` / `psql` commands when performing update queries.
    If there is any problem during the update, it is best if the query fails immediately, so you can fix the underlying problem before you execute the update again.
    If you leave this for later you risk ending up with an incompatible database, though the problems might not surface immediately.
```

>     Do not use `--force` argument for `mysql` / `psql` commands when performing update queries. If there is any problem applying the updates, it is best if the query fails immediately, so you can fix the underlying problem before trying to execute the update again. If you leave this for later, you risk ending up with an incompatible database, that might not surface before further down the road.

Author: ```suggestion
    Do not use `--force` argument for `mysql` / `psql` commands when performing update queries. If there is any problem applying the updates, it is best if the query fails immediately, so you can fix the underlying problem before trying to execute the update again. If you leave this for later you risk ending up with an incompatible database, though the problems might not surface until later.
```
